README.md

```diff
@@ -318,6 +318,36 @@ To detect the currently used localization in your tests, use the following code:
 You can access the language using the `deviceLanguage` variable.
 ```
 
+
+## Prefilling
+
+- You can prefill data within your app using [HSTestingBackchannel](https://github.com/ConfusedVorlon/HSTestingBackchannel)
+
+## Passing a Preprocessor Macro
+
+- Run snapshot using [Fastlane with .env files](https://github.com/fastlane/fastlane/blob/master/docs/FAQs.md#multiple-targets-of-the-same-underlying-app)
```

Comment:
Instead of describing how to set this up using `.env` (which is pretty advanced), I'd prefer to describe it using 

```
EXAMPLE_VALUE=1 snapshot
```

Also, `snapshot` should set `FASTLANE_SNAPSHOT=1` by default, we can easily add this to `runner.rb` and then describe in this part of the documentation, that's it's set automatically.

What do you think?